apps/fishing-map/features/download/DownloadActivityByVessel.tsx

```diff
@@ -48,7 +50,9 @@ import {
 } from './download.utils'
 
 function DownloadActivityByVessel() {
-  const { t } = useTranslation()
+  const { t, i18n } = useTranslation()
+  const { disclaimer } =
+    SOURCE_SWITCH_CONTENT[i18n.language as Locale] || SOURCE_SWITCH_CONTENT[Locale.en]
```

Comment:
isn't this cleaner ?
```suggestion
    SOURCE_SWITCH_CONTENT[i18n.language as Locale || Locale.en]
```